mino/minogrpc/server.go

```diff
@@ -400,6 +401,15 @@ func (srv *Server) Serve() error {
 	return nil
 }
 
+func (srv *Server) addNeighbour(servers ...*Server) {
+	for _, server := range servers {
+		srv.neighbours[server.addr.String()] = Peer{
```

Comment:
As mentioned earlier, at some point you will need to be able to extract the server certificate (to a file for instance), transmit it to a peer and then populate the peer server with it. From what I see, it shouldn't hard to adapt but as soon as it's done, you won't get more surprises.